bin/asc.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+const [ nodePath, thisPath, ...args ] = process.argv;
+const nodeArgs = process.execArgv;
+
+const hasSourceMaps = nodeArgs.includes("--enable-source-maps");
+const posCustomArgs = args.indexOf("--");
+const isDeno = typeof Deno !== "undefined";
+
+if (isDeno) {
+  process.on = function() { /* suppress 'not implemented' message */ };
+}
+
+if ((!hasSourceMaps || ~posCustomArgs) && !isDeno) {
+  if (!hasSourceMaps) {
+    nodeArgs.push("--enable-source-maps");
+  }
```

Comment:
why support the option if it is always enabled?

Reply:
Oh, I see, it's not AS-specific, we just always want Node to have it enabled.

Reply:
Right, previously was using an npm package for source map support, but recent Node has this option which works better. In particular, when crashing, this produces a proper trace instead of a lot of gibberish.